devices.js

```diff
@@ -46,6 +46,24 @@ const devices = {
         fromZigbee: [fz.xiaomi_battery_3v, fz.WXKG11LM_click, fz.ignore_onoff_change, fz.ignore_basic_change],
         toZigbee: [],
     },
+    'lumi.sensor_switch.aq3': {
+        model: 'WXKG12LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara wireless switch with gyro',
+        supports: 'single, double, shake, hold, release',
+        fromZigbee: [fz.xiaomi_battery_3v, fz.WXKG12LM_action_multistate,
+            fz.ignore_onoff_change, fz.ignore_basic_change, fz.ignore_multistate_change],
+        toZigbee: [],
+    },
+    'lumi.sensor_swit': {
```

Comment:
Why are there 2 entries added here, I expect to have only 1 model ID. Either lumi.sensor_switch.aq3 or lumi.sensor_swit

Reply:
I show like this in the log: 
```2018-05-28T02:45:56.536Z zigbee2mqtt:controller Recieved zigbee message with data { cid: 'genBasic', data: { modelId: 'lumi.sensor_switch.aq3' } }
2018-5-28 09:45:56 WARN Device with modelID 'lumi.sensor_swit' is not supported.
```
I can successful pair my one with lumi.sensor_swit, but i add same line for the lumi.sensor_switch.aq3, it may change with and firmware update, because there is lumi.sensor_switch.aq2 .
But if you think it not necessary you can delete the item for lumi.sensor_switch.aq3

Reply:
Reading: https://github.com/TedTolboom/com.xiaomi-mi-zigbee/issues/42 it seems that it also uses lumi.sensor_switch.aq3, lets keep both.

Reply:
`lumi.sensor_swit` seems like a strange model id to me, is this model id always reported?

Reply:
It report when press the reset button, when paired, press that button will send battery state. I have paired two of them without any problem with the code. Without lumi.sensor_swit model it could not be pair and without lumi.sensor_switch.aq3 it will not update battery. You can see the logs when i quick press reset button after it pair. This may be the bug from Xiaomi firmware or zigbeer lib. So let two model in code not a problem.

```
  zigbee2mqtt:controller Recieved zigbee message with data { cid: 'genBasic',
  data:
   { '65281':
      { '1': 2985,
        '3': 35,
        '4': 17320,
        '5': 164,
        '6': [Array],
        '8': 258,
        '10': 0,
        '152': 7,
        '153': 0 },
     modelId: 'lumi.sensor_switch.aq3' } } +13s
2018-5-29 03:07:13 INFO MQTT publish, topic: 'zigbee2mqtt/0x00158d0001d57d8b', payload: '{"battery":"97.00"}'
  zigbee2mqtt:controller Recieved zigbee message with data { cid: 'genBasic',
  data:
   { '65281':
      [ <3 empty items>,
        35,
        <2 empty items>,
        [Array],
        <145 empty items>,
        7 ],
     modelId: 'lumi.sensor_switch.aq3' } } +8ms

```